pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -169,7 +168,7 @@ type ActualStateOfWorldMounterUpdater interface {
 	MarkVolumeAsUnmounted(podName volumetypes.UniquePodName, volumeName v1.UniqueVolumeName) error
 
 	// Marks the specified volume as having been globally mounted.
-	MarkDeviceAsMounted(volumeName v1.UniqueVolumeName) error
+	MarkDeviceAsMounted(volumeName v1.UniqueVolumeName, devicePath, deviceMountPath string) error
```

Comment:
hi @jingxu, do you know why `devicePath` is changed here on the volume info? After this code change, `devicePath` in volume will be updated with new value of `WaitForAttach` func here: 
https://github.com/kubernetes/kubernetes/blob/568afb4ecca99bc3b54fddc20927b3713369d357/pkg/volume/util/operationexecutor/operation_generator.go#L488-L489
This would work on Linux node, while on Windows node, it may have issue (on windows, `devicePath` is always a Windows disk number, we cannot tell that number is from original or updated in `WaitForAttach` func), can we skip update `devicePath`  on Windows?